### DISCUSSION TITLE
Download location

### DISCUSSION THREAD

**Discussion by Lynwood** [vicwomg/pikaraoke#352] [General]
Is it possible to tell PiKaraoke where to look for karaoke files in a certain location?  I have a very large collection, and I have 2 USB sticks full, and plugged in, at the moment..I want to be able to swap out karaoke collections, and have the app look for files in those locations..  possible?

**Comment by vicwomg** [comment]
Yes, 
`./pikaraoke.sh -d <your_download_path>`

Run pikaraoke with --help to see the full list of options.

**Reply by Lynwood** [reply]
Excellent!  Would it be possible to have the program scan for available media, like if it detects a USB flash drive it would scan that for appropriate content and add it to the library if found? And if it detected a media change, rescan the drive and add the New Media to the library ?

**Reply by Lynwood** [reply]
And I just had a thought about appropriate content. 

Can they be in folders? can they be zipped?, or do they have to be all out in the root of the directory?
